src/lib/TimelineItem/TimelineItem.js

```diff
@@ -231,7 +231,7 @@ const TimelineItem = props => {
 					</div>
 				</div>
 				<Box className={props.direction === directions.LEFT ? classes.ArrowLeft : classes.ArrowRight}>
-					<Card component='div' className={classes.TimelineCard}>
+					<Card elevation={props.elevation} component='div' className={classes.TimelineCard}>
```

Comment:
@SalahAdDin Same here

Reply:
I don't know why, but if ew validate this property it does no work, but if you don't validate it, it works, also, when we don't put the property, the value would be the same as not raise: 1.